### PR TITLE
[DE] Translation Scenario-ending

### PR DESCRIPTION
**Pull request by DerSchlupf**
translated scenario-ending:
image

### CODE REVIEW

**Comment by DerSchlupf** [comment]
Same here, please wait with merge. I need some time to look at it for myself.

**Comment by qwrtln** [comment]
Move medusa a little lower, other than that OK.

**Comment by DerSchlupf** [comment]
What to do, to bring the medusa lower? In the "Heroes"-Section were other commands (node at (x,y). But here is the following:
image

**Comment by qwrtln** [comment]
It got pushed to the next page, so it won't work in this case.

**Comment by DerSchlupf** [comment]
Text is fixed, but Medusa is still very close to the text. Is it in general possible to move it lower?

**Comment by qwrtln** [comment]
`vspace*{\fill}` already moves it to the bottom of the page, so no, you can't move it lower. Let me try something else

**Comment by DerSchlupf** [comment]
Okay thanks. If it doesnt work, i can try to short 1-2 lines on the left side. But not today. Its time for bed soon^^

**Comment by DerSchlupf** [comment]
Thanks for moving the medusa. Im happy with this version and if you havent any suggestions it would be ready for merge.

**Review by qwrtln — APPROVED** [review]

File: translations/scenario_end.tex/de.po [hunk 1]
```diff
@@ -93,6 +110,14 @@ msgid ""
 "Skip steps 11--13 of Setup for the next scenario of the campaign.\n"
 "\n"
 msgstr ""
+"Nachdem du ein \\textbf{Solo-Kampagnen}-Szenario beendet hast, setze deine Heldenstufe auf 1 zurück und bereite dein Startdeck für das nächste Szenario der Kampagne wie folgt vor (anstelle der Schritte 11-13 des regulären Spielaufbaus).\n"
+"Dein Deck enthält:\n"
+"\\begin{itemize}\n"
+"  \\item Alle \\pagelink{Statistic}{Statistikkarten} deines Decks,\n"
+"  \\item Die \\pagelink{Specialty}{Sonderfähigkeitskarte} Stufe I,\n"
+"  \\item 5 weitere Karten deiner Wahl (außer Sonderfähigkeiten) aus deinem Deck .\n"
```
Comment (qwrtln) [inline translations/scenario_end.tex/de.po:118]:
```suggestion
"  \\item 5 weitere Karten deiner Wahl (außer Sonderfähigkeiten) aus deinem Deck.\n"
```

**Pull request merged**